schemas/camera_ir/dd_camera_ir.xsd

```diff
@@ -4,6 +4,143 @@
+						<coordinate1>../line_indices</coordinate1>
+					</xs:appinfo>
+				</xs:annotation>
+				<xs:complexType>
+					<xs:group ref="INT_1D"/>
+				</xs:complexType>
+			</xs:element>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:complexType name="camera_ir_frame_roi">
+		<xs:annotation>
+			<xs:documentation>Region of interest</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="name">
+				<xs:annotation>
+					<xs:documentation>Short string identifier (unique for a given device)</xs:documentation>
+					<xs:appinfo>
+						<type>dynamic</type>
+					</xs:appinfo>
+				</xs:annotation>
+				<xs:complexType>
+					<xs:group ref="STR_0D"/>
+				</xs:complexType>
+			</xs:element>
+			<xs:element name="emissivity">
+				<xs:annotation>
+					<xs:documentation>Emissivity of the region of interest</xs:documentation>
+					<xs:appinfo>
+						<type>dynamic</type>
+					</xs:appinfo>
+				</xs:annotation>
+				<xs:complexType>
+					<xs:group ref="FLT_0D"/>
+				</xs:complexType>
+			</xs:element>
+			<xs:element name="surface_area">
+				<xs:annotation>
+					<xs:documentation>Area of the region of interest</xs:documentation>
+					<xs:appinfo>
+						<type>dynamic</type>
+						<units>m^2</units>
+					</xs:appinfo>
+				</xs:annotation>
+				<xs:complexType>
+					<xs:group ref="FLT_0D"/>
+				</xs:complexType>
+			</xs:element>
+			<xs:element name="type">
+				<xs:annotation>
+					<xs:documentation>Type of the region of interest usage, physical or operational. When operational, it means machine protection is active on this region of interest</xs:documentation>
+					<xs:appinfo>
+						<type>dynamic</type>
+					</xs:appinfo>
+				</xs:annotation>
+				<xs:complexType>
+					<xs:group ref="STR_0D"/>
+				</xs:complexType>
+			</xs:element>
+			<xs:element name="polygon" type="camera_ir_frame_roi_countour" maxOccurs="unbounded">
+				<xs:annotation>
+					<xs:documentation>Set of polygons defining the region of interest</xs:documentation>
+					<xs:appinfo>
+						<coordinate1>1...N</coordinate1>
+					</xs:appinfo>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="temperature_limit_max">
+				<xs:annotation>
+					<xs:documentation>Maximum allowed temperature in the region of interest. A machine protection message is triggered if this limit is exceeded</xs:documentation>
+					<xs:appinfo>
+						<type>dynamic</type>
+						<units>K</units>
+					</xs:appinfo>
+				</xs:annotation>
+				<xs:complexType>
+					<xs:group ref="FLT_0D"/>
+				</xs:complexType>
+			</xs:element>
+			<xs:element name="temperature_max">
+				<xs:annotation>
+					<xs:documentation>Maximum temperature measured in the region of interest</xs:documentation>
+					<xs:appinfo>
+						<type>dynamic</type>
+						<units>K</units>
+					</xs:appinfo>
+				</xs:annotation>
+				<xs:complexType>
+					<xs:group ref="FLT_0D"/>
+				</xs:complexType>
+			</xs:element>
+			<xs:element name="temperature_max_line_index">
```

Comment:
Aside from the line versus row stuff, I think the order is wrong here since the quantity is the row index for the position of maximum temperature, so I think it should be:
```suggestion
			<xs:element name="row_index_temperature_max">
```